### PR TITLE
fix: wipe complex response types from node examples

### PR DESCRIPTION
**Pull request by beebzz**

### CODE REVIEW

File: src/main/java/com/twilio/oai/TwilioNodeGenerator.java [hunk 1]
```diff
@@ -250,6 +252,18 @@ private void addDependent(final Map<String, Object> dependents, final String dep
         dependent.put("filename", dependentName);
     }
 
+    private Optional<CodegenModel> resolveComplexType(Optional<CodegenModel> item) {
+        for (CodegenProperty prop: item.get().vars) {
+            if (prop.dataType.equals("Array<TestResponseObjectTestArrayOfObjects>")) {
+                prop.dataType = "Array<object>";
+            }
+            else if (prop.dataType.equals("TestResponseObjectTestObject")) {
```
Comment (childish-sambino) [inline src/main/java/com/twilio/oai/TwilioNodeGenerator.java:260]: This is too specific to the test fixture (e.g., if we change the name of the component in the fixture then this no longer works. Can we inspect the properties of the property to determine if it's a complex type?

File: src/main/java/com/twilio/oai/TwilioNodeGenerator.java [hunk 2]
```diff
@@ -250,6 +252,16 @@ private void addDependent(final Map<String, Object> dependents, final String dep
         dependent.put("filename", dependentName);
     }
 
+    private CodegenModel resolveComplexType(CodegenModel item) {
+        for (CodegenProperty prop: item.vars) {
+            if (prop.complexType != null) {
+                prop.dataType = prop.baseName.equals("test_object") ? "object" :
+                        prop.baseName.equals("test_array_of_objects") ? "Array<object>" : prop.dataType;
```
Comment (childish-sambino) [inline src/main/java/com/twilio/oai/TwilioNodeGenerator.java:259]: This is still too coupled with the naming in the test fixture. The condition needs to be based off some property that is not based on the name in the fixture.

Comment (beebzz) [reply src/main/java/com/twilio/oai/TwilioNodeGenerator.java:259]: `prop.baseType` works for the object array, but there's no helpful metadata in the `test_object` property (its base type and openapi type are still marked as `TestResponseObjectTestObject`).

Comment (childish-sambino) [reply src/main/java/com/twilio/oai/TwilioNodeGenerator.java:259]: Anything indicating it's complex?

**Review by childish-sambino — APPROVED** [review]

**Pull request merged**